### DISCUSSION TITLE
Enable folder navigation in files / image selector interface, even with a default folder

### DISCUSSION THREAD

**Discussion by julbd** [directus/directus#18587] [Feature Requests]
### Summary

As proposed by [paescuj](https://github.com/directus/directus/pull/18320#issuecomment-1525811766), add a `Folder navigation` setting to the files / image selector interface, allowing to display the folder navigation even when a default folder is set, with the following options :

- No navigation
- Limited navigation (only show sub folders)
- Full navigation (show all folders)

### Motivation

The new folder navigation introduced in 10.1.0 in hidden when a default folder is set, for consistency purpose. With this behavior, the default folder option have two different usages :

1. Setting a folder in which new files must be placed by default
2. Hiding files to others folders for privacy or simplicity purpose

These are two different objectives that should be separated.

### Alternatives

The Folder navigation could enabled everywhere, while relying on :
- For privacy concerns : permission configuration to hide some folders.
- For simplicity concerns : the folder navigation being masked under a "folders" or "arrow" icon on the side when the default folder setting is set.

### Adoption Strategy

No changes in usage if the "No navigation" option is set by default.

**Comment by vanling** [comment]
I thought this feature was not released yet.. but i always set a default folder that's why i've not seen the new folder picker as of yet.

**Comment by rijkvanzanten** [comment]
Heya! Thanks for opening this feature request!

This feature request has received over **15 votes** from the community. This means we'll move this feature request to the Under Review state! 

The Core team will schedule a meeting to review this request as soon as possible. The discussion will then be approved or denied. You may or may not be invited to join this meeting with the core team.

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by julbd** [comment]
You hou ! 🎉
(I still think the "community vote" filter is not a good filter, and misses all the small improvements nobody pays attention about but makes a product a *good* product. The 80-20 thing only apply to new features that changes the purpose of the product)
